history-service/init.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE submissions (
+	submission_id SERIAL PRIMARY KEY,
+	question_id INT NOT NULL,
+	runtime INT NOT NULL,
+	submission_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	username TEXT,
+	outcome TEXT,
```

Comment:
Outcome would be meaningful if users had test cases to run their code against (but as mentioned in another comment, we do not have that feature).